### PR TITLE
enable BraveAdblockCookieListOptIn on release channel at 10% [`production`]

### PR DESCRIPTION
**Pull request by antonok-edm**
`main`: https://github.com/brave/brave-variations/pull/407

### CODE REVIEW

**Review by bsclifton — APPROVED** [review]
++

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has quickly spot checked the change on `release` as per https://github.com/brave/brave-variations/pull/407#issuecomment-1291158245.

**Pull request merged**
